docs/DeveloperGuide.md

```diff
@@ -245,52 +245,85 @@ _{more aspects and alternatives to be added}_
-Firstly we include a Remark class under `seedu/address/Person/Model`.
+Firstly we include a Remark class under `seedu/address/model/person`.
 This class will contain all details about a remark, namely its Regex requirements and
 the string which is the remark itself.
 
 Note that a Remark field is optional when adding a Person entry into the AddressBook.
 Hence, the default value for a Remark is an empty String.
 
-There are only 2 commands which can modify the Remark for a Person. These are the 
+There are only 2 commands which can modify the Remark for a Person. These are the
 `add` command and the `edit` command.
 
 1. `add` Command (upon instantiating)
-   1. If the remark field is specified when instantiating a Person, for example:
-      `add n=JohnDoe p=98765432 e=JohnDoe@gmail.com a=Clementi r=REMARK`, their remark
-      will be REMARK. 
-   2. If not specified, it will be an empty String.
+    1. If the remark field is specified when instantiating a Person, for example:
+       `add n=JohnDoe p=98765432 e=JohnDoe@gmail.com a=Clementi r=REMARK`, their remark
+       will be REMARK.
+    2. If not specified, it will be an empty String.
 2. `edit` Command (after instantiating)
-   1. If the remark field is specified when editing a Person, for example: 
-      `edit 1 r=ChangeRemark`, their remark will be changed to ChangeRemark.
-   2. If not specified, it will remain unchanged.
+    1. If the remark field is specified when editing a Person, for example:
+       `edit 1 r=Change Remark`, their remark will be changed to ChangeRemark.
+    2. If not specified, it will remain unchanged.
 
 #### Design considerations:
 
 **Aspect: How remark can be modified:**
 
 * **Alternative 1 (current choice):** Only at `add` and `edit` command.
-    * Pros: 
-      * Reduces cluttering of commands.
-      * More intuitive design as it agrees with the definition of `add` and `edit`.
-    * Cons: 
-      * Extra typing (`r=NewRemark`) to modify the Remark field.
+    * Pros:
+        * Reduces cluttering of commands.
+        * More intuitive design as it agrees with the definition of `add` and `edit`.
+    * Cons:
+        * Extra typing (`r=New Remark`) to modify the Remark field.
 
 * **Alternative 2:** Create a `remark` command just to modify remarks.
-    * Pros: 
-      * Slightly less typing when editing Remark.
-    * Cons: 
-      * Remark is just like any other field, just like name and address.
-        Hence, it is by no mean special and does not deserve its own command.
-      * `edit` command no longer agrees with its definition to edit entries.
+    * Pros:
+        * Slightly less typing when editing Remark.
+    * Cons:
+        * Remark is just like any other field, just like name and address.
+          Hence, it is by no mean special and does not deserve its own command.
+        * `edit` command no longer agrees with its definition to edit entries.
 
 
 ### \[Proposed\] Tag command
```

Comment:
Since this feature is already implemented, should the `[proposed]` label be removed?

Reply:
I shall remove it then.